projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -807,9 +807,8 @@ function wpcom_site_has_global_styles_in_personal_plan( $blog_id = 0 ) {
 		return false;
 	}
 
-	// Placeholder experiment key, we need to update this to the new experiment key once it's created.
 	$experiment_assignment              = \ExPlat\assign_given_user( 'calypso_plans_global_styles_personal_20240127', $owner );
-	$has_global_styles_in_personal_plan = 'treatment' === $experiment_assignment;
+	$has_global_styles_in_personal_plan = null !== $experiment_assignment;
```

Comment:
Wouldn't this activate global styles on personal plan when $experiment_assignment is  `control`?
CC - @necronet

Reply:
Thanks for the explanation async that control returns null from our experimentation platform.

Reply:
No, it's the other way around. When the assignment is control we do not enable Global Styles on Personal.